affine/__init__.py

```diff
@@ -107,7 +107,7 @@ class Affine(namedtuple("Affine", ("a", "b", "c", "d", "e", "f", "g", "h", "i"))
         `a`, `b`, and `c` are the elements of the first row of the
         matrix. `d`, `e`, and `f` are the elements of the second row.
 
-    Attributes
+    Attributes:
```

Comment:
No colon here. Attributes is a numpydoc section https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring.

Reply:
It has done the same with "Notes:" too. I suppose this is awkwardly mixing google-style and numpydoc styles. I could disable this rule to keep them as is? I'm not familiar enough with google-style to reformat theses.

Reply:
Re-reading this feedback item. Perhaps the docstring style could be switched over to numpydoc? I'm way more familiar with these. I'd keep that for a separate PR though.

Reply:
It's mostly numpydoc, at least the named sections are. The extended summary and notes for `Affine` are kind of mixed together and follow Attributes. I'm not opposed to moving them upwards in a new PR.

Reply:
I'll address pydocstyle stuff in a subsequent PR.